### PR TITLE
Initial version of test job for compact cluster

### PR DESCRIPTION
**Pull request by faermanj**
Initial draft, just testing for now.

### CODE REVIEW

**Comment by mtulio** [comment]
/pj-rehearse periodic-ci-redhat-openshift-ecosystem-provider-certification-tool-main-4.14-platform-external-vsphere-compact

File: ci-operator/step-registry/opct/test/platform-external-vsphere-compact/opct-test-platform-external-vsphere-compact-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+workflow:
+  # https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40077/pull-ci-openshift-release-master-ci-operator-config/1671870759988891648
+  as: opct-test-platform-external-vsphere-compact
+  # as: opct-test-platform-external-vsphere-compact-workflow
+  steps:
+    env:
+      WORKER_REPLICAS: "0"
+      MASTER_REPLICAS: "3"
+    pre:
+      - chain: upi-vsphere-platform-external-ovn-pre
```
Comment (mtulio) [inline ci-operator/step-registry/opct/test/platform-external-vsphere-compact/opct-test-platform-external-vsphere-compact-workflow.yaml:10]: @faermanj it seems [this error](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/40077/pull-ci-openshift-release-master-ci-operator-registry/1678430377036746752) points to declare the vars in the step `upi-vsphere-platform-external-ovn-pre`. Is it correct? Are using this chain  to your setup?

In the PR I can see only the new chain `upi-conf-vsphere-platform-none-compact-ovn`, if you want to use this one, you need to define the env there.

Comment (faermanj) [reply ci-operator/step-registry/opct/test/platform-external-vsphere-compact/opct-test-platform-external-vsphere-compact-workflow.yaml:10]: This was probably copied from the reference, i'll try changing to -compact, thanks!

**Comment by faermanj** [comment]
/pj-rehearse redhat-openshift-ecosystem-provider-certification-tool-main__4.14

**Pull request closed without merging**